components/script/dom/vrframedata.rs

```diff
@@ -51,10 +51,14 @@ impl VRFrameData {
         };
 
         unsafe {
-            framedata.left_proj.set(slice_to_array_buffer_view(global.get_cx(), &matrix));
-            framedata.left_view.set(slice_to_array_buffer_view(global.get_cx(), &matrix));
-            framedata.right_proj.set(slice_to_array_buffer_view(global.get_cx(), &matrix));
-            framedata.right_view.set(slice_to_array_buffer_view(global.get_cx(), &matrix));
+            let _ = TypedArray::<Float32>::create(global.get_cx(), matrix.len() as u32, Some(&matrix),
+                framedata.left_proj.handle_mut());
+            let _ = TypedArray::<Float32>::create(global.get_cx(), matrix.len() as u32, Some(&matrix),
+                framedata.left_view.handle_mut());
+            let _ = TypedArray::<Float32>::create(global.get_cx(), matrix.len() as u32, Some(&matrix),
+                framedata.right_proj.handle_mut());
+            let _ = TypedArray::<Float32>::create(global.get_cx(), matrix.len() as u32, Some(&matrix),
+                framedata.right_view.handle_mut());
```

Comment:
nit: use `Float32Array::create` and indent the second line to match `global.get_cx()`.